src/misc/credentials/CredentialType.ts

```diff
@@ -0,0 +1,4 @@
+export const enum CredentialType {
+	internal = "internal",
+	external = "external",
```

Comment:
We should use ```PascalCase``` or ```UPPER_SNAKE_CASE``` as they seem to be our standard

Reply:
Fixed.